lib/cli/commands/getorders.ts

```diff
@@ -12,7 +12,7 @@ export const builder = {
   },
   max_results: {
```

Comment:
I think we remove it for now if it is not used.

Reply:
That's OK with me.